src/video/cocoa/cocoa_ogl.mm

```diff
@@ -185,7 +185,7 @@ - (void)viewDidChangeBackingProperties
 static FVideoDriver_CocoaOpenGL iFVideoDriver_CocoaOpenGL;
 
 
-std::optional<std::string_view> VideoDriver_CocoaOpenGL::Start(const StringList &param)
+std::optional<std::string_view> VideoDriver_CocoaOpenGL::Start(std::span<const std::string> parm)
```

Comment:
Here's what's causing the CI failure. I wonder if it's worth standardizing `param` vs `parm` as both are used in the code touched by this PR...and the latter is making me hungry for cheese. 😄